language/stdlib/modules/LibraAccount.move

```diff
@@ -971,5 +985,15 @@ module LibraAccount {
     }
 
 
+    /// # If an address has a child_vasp role, it is a vasp
+    spec module {
+        invariant [global]
+            forall child_addr1: address where Roles::spec_has_child_VASP_role_addr(child_addr1):
+                VASP::spec_is_vasp(child_addr1);
```

Comment:
If we have this global invariant, we don't need the ensures in `create_child_vasp_account` right?